kazoo/client.py

```diff
@@ -123,6 +123,9 @@ def __init__(self, hosts='127.0.0.1:2181',
             A list of authentication credentials to use for the
             connection. Should be a list of (scheme, credential)
             tuples as :meth:`add_auth` takes.
+        :param sasl_options:
```

Comment:
In order to help the user during setting, can you add the possible keys for this dict?

Reply:
done